taskipy/task_runner.py

```diff
@@ -24,8 +24,8 @@
 class TaskRunner:
     def __init__(self, cwd: Union[str, Path]):
         working_dir = cwd if isinstance(cwd, Path) else Path(cwd)
```

Comment:
I think that having two variables called `working_dir` and `self.__working_dir` (line 28) might be confusing, as they are not used for the same purpose.
Perhaps we can rename this variable into `cwd_as_path` to emphasize what it's really used for?

Reply:
Ok!

Reply:
Done on 7822e377e2aa5a65f533779770606c309760913c